java-api-core/src/com/kinvey/java/core/AbstractKinveyClientRequest.kt

```diff
@@ -49,7 +49,7 @@ protected constructor(
         /**
          * Kinvey JSON client *
          */
-        private val client: AbstractClient<BaseUser>, private val hostName: String,
+        private val client: AbstractClient<*>, private val hostName: String,
```

Comment:
What are the implications of changing the template type here? Do we have another type that `AbstractClient` can contain?